README.md

```diff
@@ -101,24 +103,16 @@ Oh, but don't say the "F" word unless the client does first, please 😜
 
 ## What has worked for us in the past
 
-We've found the following things work really well working with clients:
+Establishing trust with clients can make the difference between getting the job done and exceeding expectations, [afraser](https://github.com/afraser) has some opinions on this stuff, but here's a high-level:
```

Comment:
I would link to your soapbox instead of your profile.

Reply:
oops!